sdk/formrecognizer/ai-form-recognizer/package.json

```diff
@@ -99,9 +97,11 @@
     "@azure-tools/test-credential": "^1.0.0",
     "@azure-tools/test-recorder": "^2.0.0",
     "@microsoft/api-extractor": "7.18.11",
+    "@rollup/plugin-node-resolve": "~13.3.0",
     "@types/chai": "^4.1.6",
     "@types/mocha": "^7.0.2",
     "@types/node": "^12.0.0",
+    "@types/prettier": "~2.6.3",
     "@types/sinon": "^9.0.4",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
```

Comment:
esm devDep is no longer needed now that dev-tool handles its functionality?

Reply:
Good point!

Reply:
reverted this change